model/src/main/java/com/cloudera/nav/sdk/model/relations/Relation.java

```diff
@@ -178,11 +195,11 @@ public T ep2(Collection<Entity> ep2Entities) {
   private String identity;
   @MProperty(required=true)
   private RelationType type;
-  @MProperty(required=true, attribute="endpoint1Ids")
```

Comment:
Will need to add a validation saying that either the ep1Id or ep1Attribtues, one of these must be present for the Relation to be valid.